update-whatprovides-db.sh

```diff
@@ -7,7 +7,9 @@
 set -e -u
 
 list_files() {
-	dpkg-deb -c "${1}" | grep -o "/data/data/com.termux/.*" | sed -E 's@(.*) ->..*@\1@g'
+	dpkg-deb -c "${1}" | grep -o "/data/data/com\.termux/files/.\+" \
+		| sed -E 's@(.*) ->..*@\1@g;s@/$@@g' \
```

Comment:
note the change in the `sed` command is required even without merging this pr @xeffyr

Reply:
Ok